df.entities.xml

```diff
@@ -2,22 +2,22 @@
     <struct-type type-name='entity_occasion_info'>
         <stl-vector name="occasions" pointer-type='entity_occasion' since='v0.42.01'/>
         <int32_t name="next_occasion_id"/>
-        <static-array type-name='int32_t' count='10'/>
-        <int32_t />
+        <static-array name='events' type-name='int32_t' count='10' ref-target='history_event'/>
+        <int32_t name='count' comment="number of elements used in array above"/>
     </struct-type>
-    <struct-type type-name='entity_occasion'>
+    <struct-type type-name='entity_occasion' comment="some festivals are annual, some single time. anon_1=0 plus anon_3=0 seems to match with single time, which doesn't make much sense. Only frequency seen is yearly">
         <int32_t name="id"/>
-        <int32_t />
-        <int32_t />
-        <int32_t />
+        <int32_t comment="0 and 1 seen"/>
+        <int32_t name='site' ref-target='world_site'/>
+        <int32_t init-value='-1' comment="only -1 seen, but based on other cases, might be an abstract building"/>
         <compound name='name' type-name='language_name'/>
-        <int32_t />
-        <int32_t />
-        <int32_t />
-        <int32_t name="event"/>
-        <int32_t />
-        <stl-vector name='schedule' pointer-type='entity_occasion_schedule' />
-        <int32_t />
+        <int32_t name='start_year_tick'/>
+        <int32_t name='end_year_tick'/>
+        <int32_t comment="0-2 seen"/>
```

Comment:
In the future, it may be worth making these sorts of fields enums with `Unk` items

Reply:
Here I disagree. I want more evidence to indicate that it actually is an enum rather than e.g. an id/index of some local structure.